docs/reference/tmx-map-format.rst

```diff
@@ -515,9 +533,9 @@ properties from the specified template, properties saved with the object
 will have higher priority, i.e. they will override the template
 properties.
 
-Can contain: :ref:`tmx-properties`, :ref:`tmx-ellipse` (since
+Can contain at most one: :ref:`tmx-properties`, :ref:`tmx-ellipse` (since
 0.9), :ref:`tmx-point`, :ref:`tmx-polygon`, :ref:`tmx-polyline`, :ref:`tmx-text`
-(since 1.0), *image*
+(since 1.0), :ref:`tmx-image`
```

Comment:
The image element was included in italics because it used to be supported by the old Java version of Tiled, but the current version of Tiled never supported that. It should probably just be removed since it's unlikely to be relevant for anybody.
```suggestion
(since 1.0)
```